src/engine/server/server.cpp

```diff
@@ -1506,6 +1516,44 @@ void CServer::ConMapReload(IConsole::IResult *pResult, void *pUser)
 	((CServer *)pUser)->m_MapReload = 1;
 }
 
+void CServer::ConSaveConfig(IConsole::IResult *pResult, void *pUser)
+{
+	CServer *pThis = ((CServer *)pUser);
+	IConfig* currentConfig = pThis->Kernel()->RequestInterface<IConfig>();
```

Comment:
`pCurrentConfig` instead of `currentConfig` (following https://www.teeworlds.com/?page=docs&wiki=nomenclature)